layouts/_default/baseof.html

```diff
@@ -1,6 +1,9 @@
 <!doctype html>
 <html lang="{{ .Site.Language.Lang }}" class="{{.Params.class}} no-js">
-  <head>
+{{- if eq hugo.Environment "preview" -}}
+  <!-- deploy preview -->
```

Comment:
Is this clause needed?

Reply:
It helps us check we're seeing a deploy preview build.

Reply:
(the comment should appear in the published HTML, but doesn't render into the page)

Reply:
Ah, okay.